final/src/main/java/ua/epam/spring/hometask/controller/rest/UserRestController.java

```diff
@@ -0,0 +1,58 @@
+package ua.epam.spring.hometask.controller.rest;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ua.epam.spring.hometask.dto.UserDTO;
+import ua.epam.spring.hometask.facade.UserFacade;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/api/users")
+public class UserRestController {
+
+    @Autowired
+    private UserFacade userFacade;
+
+    @ResponseBody
+    @PostMapping("/create")
```

Comment:
Bear in mind, @RequestMapping annotation is still widely used, so however you will need to use it instead of @PostMapping/@GetMapping to fulfill the consistency of the code style used on the project.